jenkinsfiles/Jenkinsfile.search-reindex

```diff
@@ -11,4 +11,4 @@ elifePipeline({
             builderCmdNode 'search--prod', 1, "cd /srv/search; ./bin/reindex elife_search_reindex_${env.BUILD_NUMBER}"
         }
     }
-}, timeoutInMinutes=240)
```

Comment:
This is were it was used:
https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/elifePipeline.groovy#L63
Unclear why it provoked a 0-minute timeout. This should assign (accidentally) a variable called `timeoutInMinutes` and then pass in the value nonetheless.